lib/taste_tester/commands.rb

```diff
@@ -210,5 +210,200 @@ def self.upload
       logger.error(exception.backtrace.join("\n"))
       exit 1
     end
+
+    def self.impact
+      # Use the repository specified in config.rb to calculate the changes
+      # that may affect Chef. These changes will be further analyzed to
+      # determine specific roles which may change due to modifed dependencies.
+      repo = BetweenMeals::Repo.get(
+        TasteTester::Config.repo_type,
+        TasteTester::Config.repo,
+        logger,
+      )
+      if repo && !repo.exists?
+        fail "Could not open repo from #{TasteTester::Config.repo}"
+      end
+
+      changes = _find_changeset(repo)
+
+      # Use Knife (or custom logic) to check the dependencies of each role
+      # against the list of changes. `impact_roles` will contian the set
+      # of roles with direct or indirect (dependency) modifications.
+      if TasteTester::Config.use_custom_impact_hook
+        impact_roles = TasteTester::Hooks.custom_impact(changes)
+      else
+        impact_roles = _find_impact(changes)
+      end
+
+      # Do any post processing required on the list of impacted roles, such
+      # as looking up hostnames associated with each role.
+      unless TasteTester::Config.skip_post_impact_hook
```

Comment:
I don't think this warrants a flag. The reason some hooks have such a flag is because they're primary validation hooks which someone might need to bypass, but in this case it doesn't make sense and having this combination of "use this" and "don't use this other part even though I forcefully enabled a related part" It hink is just confusing.

Reply:
Agreed. I've removed the flag and `post_impact` will always run and will just be a NOP by default.